catalog/nsili/catalog-nsili-source/src/main/java/org/codice/alliance/nsili/source/NsiliSource.java

```diff
@@ -420,17 +419,6 @@ private void getIorStringFromSource() {
      */
     private void initLibrary() {
         if (iorString != null) {
-
-            if (LOGGER.isDebugEnabled()) {
-                try {
-                    //Log the IOR content for debugging purposes
-                    IOR ior = new IOR(iorString.getBytes());
```

Comment:
No. It's not really able to parse all of the IOR's we have seen, so will need to find something else to use. I have removed it.